src/test/java/software/amazon/awssdk/crt/test/HttpRequestResponseTest.java

```diff
@@ -25,19 +25,12 @@
 import java.util.concurrent.TimeUnit;
 
 public class HttpRequestResponseTest extends HttpRequestResponseFixture {
-    private final static String HOST = "https://postman-echo.com";
+    private final static String HOST = "https://localhost:8082";
```

Comment:
let's put a comment to point this to the readme of the local host in aws-c-http, so people know where this comes from, and how to use it.

https://github.com/awslabs/aws-c-http/tree/v0.10.7/tests/mock_server

ditto to other places we use the local host.